lib/coins/via.js

```diff
@@ -0,0 +1,52 @@
+// https://github.com/viacoin/viacoin/blob/master/src/chainparams.cpp
+
+var common = {
+  name: 'Viacoin',
+  unit: 'VIA'
+}
+
+var main = Object.assign({}, {
+  hashGenesisBlock: '4e9b54001f9976049830128ec0331515eaabe35a70970d79971da1539a400ba1',
+  port: 5223,
+  protocol: {
+    magic: 0xcbc6680f
+  },
+  seedsDns: [
+    'seed.viacoin.net',
+    'viaseeder.barbatos.fr',
+    'mainnet.viacoin.net'
+  ],
+  versions: {
+    bip32: {
+      private: 0x05358394,
+      public: 0x0488b21e
+    },
+    bip44: 14,
+    private: 0xc7,
+    public: 0x30,
+    scripthash: 0x47
```

Comment:
Can you point from where you take this constants? They are not match: https://github.com/viacoin/viacoin/blob/04672d3d0a696606df0933bb307709fa23235eb4/src/chainparams.cpp#L137-L141

Reply:
Must have been confused. Just forced pushed an update.